projection/core/src/main/scala/com/lightbend/lagom/internal/projection/ProjectionRegistryActor.scala

```diff
@@ -137,12 +136,29 @@ class ProjectionRegistryActor extends Actor with ActorLogging {
         case None             => log.error(s"Unknown actor [${sender().path.toString}] reports status $observedStatus.")
       }
 
-    // TODO: handle UpdateSuccess/UpdateFailure !!
-    case UpdateSuccess(_, _) => //TODO
-    case _: UpdateFailure[_] => //TODO
+    case UpdateSuccess(_, _) => //noop: the update op worked nicely, nothing to see here
+
+    // There's three types of UpdateFailure and 3 target CRDTs totalling 9 possible cases of
+    // which only UpdateTimeout(_,_) is relevant.
+    // case UpdateTimeout(ObservedStatusDataKey, _) =>
+    //    the observed status changes very rarely, but when it changes it may change multiple times in a short
+    //    period. The fast/often changes probably happen on a cluster rollup, up/down-scale, etc... In any case,
+    //    data eventually will become stable (unchanging)in which case data is eventually gossiped and last writer wins.
+    // case UpdateTimeout(RequestedStatusDataKey, _) =>
+    //    the request status changes very rarely. It is safe to ignore timeouts when using WriteMajority because
+    //    data is eventually gossiped.
+    // case UpdateTimeout(NameIndexDataKey, _) =>
+    //    the data in the nameIndex is only-grow until reaching a full hardcoded representation. It is safe to
+    //    ignore timeouts when using WriteMajority because data is eventually gossiped.
+    // In any other UpdateFailure cases, noop:
+    // - ModifyFailure: using LWWMap with `put` as the modify operation will never fail, latest wins.
+    // - StoreFailure: doesn't apply because we don't use durable CRDTs
+    case _: UpdateFailure[_] =>
```

Comment:
The code is effectively the same but includes a long comment on why both cases are `noop`.